### DISCUSSION TITLE
Adafactor optimizer work?

### DISCUSSION THREAD

**Discussion by toriving** [deepspeedai/DeepSpeed#2480] [Q&A]
hello.

I am training a model using Deepspeed.
I tried using the adafactor (non-native) as an optimizer.
However, it was confirmed that the loss did not decrease, and this phenomenon only occurs when using Deepspeed.

Doesn't Deepspeed currently support the Adaptive optimizer style?
Or do it not support Adafactor?

**Comment by tjruwase** [comment]
@toriving, thanks for your question. Can you please create an issue so we can investigate further?
